cmd/clusterctl/client/cluster/objectgraph.go

```diff
@@ -226,15 +257,26 @@ func (o *objectGraph) objToNode(obj *unstructured.Unstructured) *node {
 			Name:       obj.GetName(),
 			Namespace:  obj.GetNamespace(),
 		},
-		owners:     make(map[*node]ownerReferenceAttributes),
-		softOwners: make(map[*node]empty),
-		tenant:     make(map[*node]empty),
-		virtual:    false,
+		owners:         make(map[*node]ownerReferenceAttributes),
+		softOwners:     make(map[*node]empty),
+		tenant:         make(map[*node]empty),
+		virtual:        false,
+		additionalInfo: make(map[string]interface{}),
+	}
+	if err := o.objInfoToNode(obj, newNode); err != nil {
+		return nil, errors.Wrap(err, "failed to add object into to node")
```

Comment:
what about adding kind/name of the obj in errors
(same for other errors added in this PR)